spikeextractors/extractors/matsortingextractor/matsortingextractor.py

```diff
@@ -67,3 +75,6 @@ def _drill(d: dict, keys: deque):
         else:
             return self._data[fieldname][()]
 
+    @staticmethod
+    def write_dict_v7_3(mat_file_path, dict_to_write): # field must be a dict
```

Comment:
would it make sense to have a `write_dict_to_mat()` function with a `version` argument? 

If >= 7.3 --> hdf5storage, otherwise scipy.io.savemat

@rdiggelmann what do you think?